purchase_stock_picking_return_invoicing/models/purchase.py

```diff
@@ -147,18 +145,29 @@ def _compute_qty_returned(self):
             ["purchase_line_id", "product_uom"],
             lazy=False,
         )
-        p = self._prefetch
```

Comment:
@pedrobaeza 
AFIK, the prefetching is changed. I am not sure what I did here is correct.